### PR TITLE
AL-2741 Add retry for AWP tests

### PR DESCRIPTION
**Pull request by batyakopelowitz**

### CODE REVIEW

File: vendor/github.com/dome9/dome9-sdk-go/services/awp/onboarding_common.go [hunk 1]
```diff
@@ -61,64 +61,82 @@ type GetAWPOnboardingResponse struct {
...
+	return nil, fmt.Errorf("request failed after %d attempts: %w", maxRetries, err)
+}
+
+
+func CreateAWPOnboarding(client *client.Client, req interface{}, path string, queryParams CreateOptions) (*http.Response, error) {
+	maxRetries := 3
+	retryInterval := time.Second * 5
+
+	f := func() (*http.Response, error) {
+		return client.NewRequestDo("POST", path, queryParams, req, nil)
```
Comment (chkp-dolevh) [inline vendor/github.com/dome9/dome9-sdk-go/services/awp/onboarding_common.go]: pls use NewRequestDoRetry func.
example:
image

**Review by chkp-dolevh — CHANGES_REQUESTED** [review]
note the note

File: vendor/github.com/dome9/dome9-sdk-go/services/awp/onboarding_common.go (hunk 1 above)
Comment (batyakopelowitz) [reply vendor/github.com/dome9/dome9-sdk-go/services/awp/onboarding_common.go]: done

**Review by chkp-shayderi — APPROVED** [review]
LGTM

**Review by chkp-dolevh — APPROVED** [review]

**Pull request merged**